### ISSUE TITLE
Access is not a protocol

### ISSUE THREAD

**Issue by arnaud-lb** [SenecaSystems/red_black_tree#10]
Compiling lib/red_black_tree.ex results in the following error with Elixir 1.4:

```
** (ArgumentError) Access is not a protocol
    (elixir) lib/protocol.ex:76: Protocol.assert_protocol!/2
    lib/red_black_tree.ex:623: (file)
    (elixir) lib/kernel/parallel_compiler.ex:117: anonymous fn/4 in Kernel.ParallelCompiler.spawn_compilers/1
```

**Comment by cschneid** [comment]
@tyre I have run into this myself. If I were to update the package to work with Elixir 1.4, would you be around  to merge a fix?

**Comment by ndemonner** [comment]
@cschneid yeah we can merge it, thanks

**Comment by ndemonner** [comment]
Fixed in #11. Thanks @cschneid!
